go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -459,6 +459,16 @@ func (vttablet *VttabletProcess) QueryTablet(query string, keyspace string, useD
 	return executeQuery(conn, query)
 }
 
+// MultiQueryTablet lets you execute a query in this tablet and get the result
+func (vttablet *VttabletProcess) MultiQueryTablet(sql string, keyspace string, useDb bool) error {
```

Comment:
Same here, this is not used yet. I am guessing there will be a follow-up PR to actually use it?

Reply:
That's correct -- there's a followup coming.